spec/approvals/cli/generate/ruby-code

```diff
@@ -0,0 +1,14 @@
+require "victor"
```

Comment:
Can these fixture files have `.rb` extensions so that highlighting works as expected?

Reply:
Yes. Done.